typings/plugins.d.ts

```diff
@@ -0,0 +1,163 @@
+import * as Blockly from "./blockly";
```

Comment:
Please note: This file does not represent what is implemented so far, it's just a potential API.